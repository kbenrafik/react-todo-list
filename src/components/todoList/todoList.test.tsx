import React from "react";
import { render, screen } from "@testing-library/react";
import TodoList from "./todoList";

test("renders TodoList", () => {
  const mockProps = {
    items: [
      {
        title: "todo 1",
      },
      {
        title: "todo 2",
      },
    ],
  };

  const onAddItemMock = () => {}
  const component = render(<TodoList items={mockProps.items} onAddItem={onAddItemMock} />);

  expect(component.container).toMatchSnapshot();
});
