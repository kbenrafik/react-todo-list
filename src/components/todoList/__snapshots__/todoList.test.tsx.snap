// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TodoList 1`] = `
<div>
  <div
    class="todo-list"
  >
    <div
      class="todo-item"
    >
      <span>
        1
        . 
      </span>
      <input
        placeholder="Title"
        type="text"
        value="todo 1"
      />
    </div>
    <div
      class="todo-item"
    >
      <span>
        2
        . 
      </span>
      <input
        placeholder="Title"
        type="text"
        value="todo 2"
      />
    </div>
  </div>
</div>
`;
